nginx/ng/install.sls

```diff
@@ -3,14 +3,20 @@
 # Manages installation of nginx.
 
 {% from 'nginx/ng/map.jinja' import nginx, sls_block with context %}
+{% set nginx_package = nginx.lookup.package %}
+{% if salt['grains.get']('os_family') == 'Debian' %}
```

Comment:
Is there no way to do this in the map.jinja? That's what the maps are meant for (so you don't have a bunch of grains lookups all over every state).